refinery/ui/source/js/commons/partials/download-files-button.html

```diff
@@ -0,0 +1,43 @@
+<button
+    class="refinery-base btn btn-default btn-xs header-button"
+    data-toggle="modal" data-target="#multi-file-download-modal">
+    Download Files
+</button>
+<span class="p-l-1-3">
+  <a
+    popover-placement="left"
+    uib-popover-html="'Display a cURL command that will allow
+    you to download the current set of filtered files from the command line.'"
+    popover-title="Download Files"
+    popover-trigger="'outsideClick'"
+    popover-append-to-body="true">
+      <i class="fa fa-question-circle info-icon icon-only"></i>
+  </a>
+</span>
+<div class="modal fade" id="multi-file-download-modal" tabindex="-1" role="dialog">
```

Comment:
In the recent past, I've separated out the modals from their triggers (buttons). It makes for better usability (smaller and focused components). Other elements could trigger the modal for example. Not a deal breaker.